docs/synapse/devguides/index_cells.rst

```diff
@@ -0,0 +1,15 @@
+
+Synapse Devops
+##############
+
+The DevOps guide contains information useful for running **Synapse** ecosystem components. This will continue to be
+updated and expanded over time.
```

Comment:
strike "This will continue to be updated" 

Of course it is

Reply:
consistent language with other parts of the documentation.